BritneyS/DnDCharacterGenerator
# Apollo graphql setup

## What does this PR do :question:
This PR sets up Apollo iOS in order for the iOS client to be able to request resources from a GraphQL server ([This one!](https://github.com/BritneyS/dnd-graphql)).


## How to test it :microscope:
1. Checkout this branch
2. Run `pod install`
3. Run the app, and note that it builds and runs successfully (simulator screen should be blank).



## Any background info you would like to include :white_check_mark:
Apollo iOS installation instructions are [here](https://www.apollographql.com/docs/ios/installation).


## Screenshots (if applicable) :camera:
N/A
